packages/kit/src/runtime/app/forms.js

```diff
@@ -130,6 +130,11 @@ export function enhance(form_element, submit = () => {}) {
 
 	/** @param {SubmitEvent} event */
 	async function handle_submit(event) {
+		const method = event.submitter?.hasAttribute('formmethod')
+			? /** @type {HTMLButtonElement | HTMLInputElement} */ (event.submitter).formMethod
+			: clone(form_element).method;
```

Comment:
We also could simplify this since we are already checking what the `method` of the form element is [here](https://github.com/sveltejs/kit/blob/b6bb5e8d17224da00c05104cdf96f630f1c3f47d/packages/kit/src/runtime/app/forms.js#L100)

The only downside is that it would still cause issues outside of DEV or if the form's method is changed at runtime - so it's probably not worth it